_drafts/2017-09-27-new-sn7-components.md

```diff
@@ -0,0 +1,135 @@
+---
+​---
+title: "Entering the new era - the new, componentized SN7"
+author: borsi 
+image: "../img/posts/components.jpg"
+tags: [sn7, components, componentization]
+​---
+---
+
+It's been a long one year, but we're nearing the end of the componentization project for sensenet ECM. If you tried to keep up with the progress, what you could see, was a bunch of git repositories, NuGet and npm packages popping up now and again, but you may not be sure what each of them really does. You might be thinking "Okay, you tore the whole thing to pieces, but how the hell does it all fit together?" Fear not.
+
+I’ll try and break down these components to you, so when you accidentally hop on to our github enterprise page, you won’t feel lost in the forest of repositories.
+
+*[[DISCLAIMER: I am not coming up with all this all by myself. People smarter than me, have already written a great description about this [here](http://community.sensenet.com/docs/sensenet-components/). So if you don't like my style, by all means, hop over there and read everything you can find.]]*
+
+If you're still here, fasten your seatbelts. It's going to be a long ride. So what's lurking under the SenseNet enterprise in GitHub?
+
+
+
+## The Core
+
+*The essentials. The alpha, and the omega. What you need to have a powerful content management platform up and running on your premises.*
+
+[**SenseNet/sensenet**](https://github.com/SenseNet/sensenet)
+
+Current NuGet package: <https://www.nuget.org/packages/SenseNet.Services.Install/7.0.0-beta4>
```

Comment:
I think we should remove version postfix from nuget links so they always point to the latest version, even if we publish new versions in the near future.

Reply:
(or we can use those beautiful badges that display the current package version dynamically)